src/Transport/Seam/MessagePump.cs

```diff
@@ -20,6 +20,7 @@ class MessagePump : IPushMessages, IDisposable
         ILog logger = LogManager.GetLogger(typeof(MessagePump));
         string inputQueue;
         SatelliteTransportAddressCollection satelliteTransportAddresses;
+        SemaphoreSlim throttler = new SemaphoreSlim(1);
```

Comment:
Not needed since will be re-assigned in [`OnStart()`](https://github.com/Particular/NServiceBus.AzureServiceBus/pull/354/files#diff-f62072a55a1af73f0df134b65d530d8cR103)